src/time.rs

```diff
@@ -212,7 +212,7 @@ impl std::fmt::Display for ClockId {
 
 /// Get the resolution of the specified clock, (see
 /// [clock_getres(2)](https://pubs.opengroup.org/onlinepubs/7908799/xsh/clock_getres.html)).
-#[cfg(not(target_os = "redox"))]
+#[cfg(not(any(target_os = "redox", target_os = "haiku")))]
 #[cfg_attr(docsrs, doc(cfg(all())))]
 pub fn clock_getres(clock_id: ClockId) -> Result<TimeSpec> {
```

Comment:
Haiku seems to support this.

Reply:
i think this is a similar situation as above, haiku does support it but seems like libc does not